Demo/Sources/YouTube/YoutubeIdentifier/Sources/YouTubeIdentifier/YouTubeIdentifier.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+import Foundation
+
+/// A namespace for YouTube identifier extraction utilities.
+public enum YouTubeIdentifier {
+    /// Extracts the YouTube identifier from the given URL.
+    /// - Parameter url: The URL from which to extract the YouTube identifier.
+    /// The extracted YouTube identifier if found, or `nil` if no identifier could be extracted.
+    /// - Returns: The extracted YouTube identifier if found, or `nil` if no identifier could be extracted.
```

Comment:
```suggestion
    /// Extracts the YouTube identifier from the given URL.
    ///
    /// - Parameter url: The URL from which to extract the YouTube identifier. The extracted YouTube identifier if found, or `nil` if no identifier could be extracted.
    /// - Returns: The extracted YouTube identifier if found, or `nil` if no identifier could be extracted.
```

Reply:
It was my first version, but the linter was not happy with that. 
Indeed, the limit is set to 160 characters.